packages/rs-platform-version/src/version/mocks/v3_test.rs

```diff
@@ -897,6 +897,7 @@ pub(crate) const TEST_PLATFORM_V3: PlatformVersion = PlatformVersion {
             },
         },
         state_transitions: StateTransitionVersions {
+            max_state_transition_size: 20000,
```

Comment:
How does this relate to tenderdash transaction size? Should we also set Tenderdash `max-tx-bytes` accordingly?

Reply:
The Tenderdash max-tx-bytes is the relay size. I believe we should also reduce it.